auto-check-moved-or-rename/check_moved_or_rename.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+input_file="test.txt"
+output_file="output.txt"
+
+if [ ! -f "$input_file" ]; then
+    echo "Error: File '$input_file' does not exist."
+    exit 1
+fi
+
+> "$output_file"
+
+while IFS= read -r line || [[ -n "$line" ]]; do
+    echo "Processing line: $line" | tee -a "$output_file"
+
+    IFS=',' read -r project_url sha module_path package_class_path _ <<< "$line"
+
+    class_name=$(echo "$package_class_path" | awk -F '.' '{print $(NF-1)}')
+    package_path=$(echo "$package_class_path" | awk -F '.' '{for (i=1; i<NF-1; i++) printf "%s/", $i; print ""}')
+    current_file_path="$module_path/src/test/java/$package_path$class_name.java"
```

Comment:
This structure may not be correct always but it is a good starting point and students can modify it if and when required